components/SummaryContainer.module.css

```diff
@@ -241,4 +216,4 @@ div.btnsContainer button span {
 div.btnsContainer button:hover {
   cursor: pointer;
   background-color: lightgray;
-}
\ No newline at end of file
+}
```

Comment:
I'm assuming most of this file will be double checked/cleaned up as part of the card for refactoring the css in this sprint?